templates/main/login.html

```diff
@@ -7,6 +7,20 @@
         <script src="js/login.js" defer></script>
     </head>
     <body>
+    	<div id='container'>
         <h1>Login page</h1>
+    	 <form action="" method="POST">
+			  
+			  	
+			    <label for="uname"><b>Username</b></label>
+			    <input type="text" placeholder="Enter Username" name="uname" required>
+			    <br>
+			    <label for="psw"><b>Password</b></label>
+			    <input type="password" placeholder="Enter Password" name="psw" required>
+			    <br>
+			    <button type="submit" name='submitbutton'>Login</button>
+			
```

Comment:
You indentation and line spacing is inconsistent, this may create hard-to-read code.